examples/src/main/java/org/apache/spark/examples/sql/hive/JavaSparkHiveExample.java

```diff
@@ -124,6 +124,13 @@ public static void main(String[] args) {
     // ...
     // $example off:spark_hive$
 
+    // Hive serde's are also supported with serde properties.
+	String sqlQuery = "CREATE TABLE src_serde(key decimal(38,18), value int) USING hive"
```

Comment:
Hi, @crlalam.
We use 2-space indentation in general.
FYI, maybe, you can see [Scala Coding Style](https://github.com/databricks/scala-style-guide).